src/devices/xiaomi.ts

```diff
@@ -814,8 +814,16 @@ const definitions: Definition[] = [
         model: 'ZNXDD01LM',
         vendor: 'Xiaomi',
         description: 'Aqara ceiling light L1-350',
-        extend: xiaomiExtend.light_onoff_brightness_colortemp({disableEffect: true, colorTempRange: [153, 370]}),
-        ota: ota.zigbeeOTA,
+        toZigbee: xiaomiExtend.light_onoff_brightness_colortemp({
+            disableEffect: true, disablePowerOnBehavior: true, disableColorTempStartup: true,
+        }).toZigbee.concat([tz.xiaomi_switch_power_outage_memory]),
+        fromZigbee: xiaomiExtend.light_onoff_brightness_colortemp({
+            disableEffect: true, disablePowerOnBehavior: true, disableColorTempStartup: true,
+        }).fromZigbee.concat([fz.aqara_opple]),
+        exposes: xiaomiExtend.light_onoff_brightness_colortemp({
+            disableEffect: true, disablePowerOnBehavior: true, disableColorTempStartup: true,
+            colorTempRange: [153, 370],
+        }).exposes.concat([e.power_outage_memory(), e.device_temperature(), e.power_outage_count()]),
```

Comment:
I know you can't test it, but I expect that all the other Xiaomi lights work the same, there I suggest to modify the `xiaomiExtend` instead

Reply:
ok, made an extension of `light_onoff_brightness_colortemp` options, modified the `light_onoff_brightness_colortemp` itself and applied it to all other xiaomi devices

Reply:
Oh, I forget to add other devices to aqaraOpple converter. Will push a new commit soon...